src/lib.rs

```diff
@@ -537,9 +537,73 @@ function x() {
         eprintln!("{}", collection.as_ref().unwrap_err().message);
         assert_eq!(
             collection.unwrap_err().message,
-            r#"Error found at line 3, column 22: Unrecognized token "object". Expected one of: "number", "string"
+            r#"Error found at line 3, column 22: Unrecognized token "object". Expected one of: "map", "number", "string"
```

Comment:
Should we not include array in this?

Reply:
Array is parsed as 3 tokens, ("number" OR "string") + "[" + "]", the message is generated by Lalrpop and it only shows the next expected token